webtau-junit5-examples/src/test/groovy/com/example/tests/junit5/DynamicTestsGroovyTest.groovy

```diff
@@ -13,4 +13,16 @@ class DynamicTestsGroovyTest {
             PriceCalculator.calculate(price, quantity).should == outcome
         }
     }
+
+    @TestFactory
+    def "individual tests can use an optional display label to clarify the use case"() {
+        ["label",           "price", "quantity", "outcome"] {
+        ___________________________________________________
+          "positive price" | 10     |  30      |  300
+          "negative price" | -10    |  30      | -300
+          null             | 0      |  30      | 0
```

Comment:
Let’s separate bull case into a separate test. Reason is we should use this test method body in docs

Reply:
done, and added this file in the docs